### PR TITLE
BRUCE-6697: Cleanup logs and add debug logs in manager service

### PR DESCRIPTION
**Pull request by kavyamani596**

### CODE REVIEW

File: svc-managers/managers/managers.go [hunk 1]
```diff
@@ -501,7 +513,7 @@ func fillResponse(ctx context.Context, body []byte, managerData map[string]inter
 	var respData map[string]interface{}
 	err := json.Unmarshal([]byte(data), &respData)
 	if err != nil {
-		l.LogWithFields(ctx).Error(err.Error())
+		l.LogWithFields(ctx).Error("unable to unmarshal : " + err.Error())
```
Comment (chauberahul1993) [inline svc-managers/managers/managers.go]: err.Error() also give same error , no need to additional message

File: svc-managers/managers/managers.go [hunk 2]
```diff
@@ -566,11 +580,13 @@ func (e *ExternalInterface) GetRemoteAccountService(ctx context.Context, req *ma
 	resp.Body = resource
 	resp.StatusCode = http.StatusOK
 	resp.StatusMessage = response.Success
+	respBody := fmt.Sprintf("%v",resp.Body)
+	l.LogWithFields(ctx).Debugf("outgoing remote account service response to northbound: %v", string(respBody))
```
Comment (chauberahul1993) [inline svc-managers/managers/managers.go]: %v should replace with %s

**Review by chauberahul1993 — COMMENTED** [review]
Please replce %v to %s all place where required

**Review by chauberahul1993 — APPROVED** [review]

File: svc-managers/managers/managers.go (hunk 1 above)
Comment (kavyamani596) [reply svc-managers/managers/managers.go]: done

**Review by amar-shalgar — APPROVED** [review]

**Pull request merged**